client/beefy/src/worker.rs

```diff
@@ -482,7 +495,14 @@ where
 			)?,
 			RoundAction::Enqueue => {
 				debug!(target: "beefy", "🥩 Buffer vote for round: {:?}.", block_num);
-				self.pending_votes.entry(block_num).or_default().push(vote)
+				if self.pending_votes.len() < MAX_BUFFERED_VOTE_ROUNDS as usize {
```

Comment:
make `MAX_BUFFERED_VOTE_ROUNDS` be `usize` and avoid this cast (same for justifs max)

```suggestion
				if self.pending_votes.len() < MAX_BUFFERED_VOTE_ROUNDS {
```